app/models/publication.rb

```diff
@@ -421,14 +418,7 @@ def doi_url_path
         end
         field(:doi_verified)
       end
-      group :open_access_permissions do
-        field(:preferred_version)
-        field(:set_statement)
-        field(:checked_for_set_statement)
-        field(:licence)
-        field(:embargo_date)
-        field(:checked_for_embargo_date)
-      end
+      field(:preferred_version)
```

Comment:
I think this is fine, but it might be a bit more user friendly if the :preferred_version was down at the bottom near the DOI grouping.  Just so all the AI OA Workflow stuff is close together.